.clang-format

```diff
@@ -0,0 +1,27 @@
+BasedOnStyle: LLVM
+IndentWidth: 4
+AllowShortIfStatementsOnASingleLine: false
+AlwaysBreakTemplateDeclarations: Yes
+BinPackArguments: false
+BinPackParameters: false
+BreakBeforeBraces: Attach
+ColumnLimit: 120
+ConstructorInitializerIndentWidth: 4
+Cpp11BracedListStyle: true
+DerivePointerAlignment: false
+IncludeBlocks: Regroup
+KeepEmptyLinesAtTheStartOfBlocks: false
+NamespaceIndentation: None
+PointerAlignment: Right
+ReflowComments: true
+SortIncludes: false
+SpaceAfterCStyleCast: false
+SpaceBeforeAssignmentOperators: true
+SpaceBeforeParens: ControlStatements
+SpaceInEmptyParentheses: false
+SpacesBeforeTrailingComments: 1
+SpacesInAngles: Never
+SpacesInCStyleCastParentheses: false
+SpacesInParentheses: false
+SpacesInSquareBrackets: false
+AlignEscapedNewlines: Right
```

Comment:
The following options are already set to the values here in clang-format 15:

- BreakBeforeBraces
- ConstructorInitializerIndentWidth
- Cpp11BracedListStyle
- NamespaceIndentation
- ReflowComments
- SpaceAfterCStyleCast
- SpaceBeforeAssignmentOperators
- SpaceBeforeParens
- SpaceInEmptyParentheses
- SpacesInAngles
- SpacesInCStyleCastParentheses
- SpacesInParentheses
- SpacesInSquareBrackets
- AlignEscapedNewlines
Should they be specified here? I find it redundant.

Reply:
Removed redundant options